smarts/core/controllers/lane_following_controller.py

```diff
@@ -78,8 +78,9 @@ def perform_lane_following(
         state = controller_state
         # This lookahead value is coupled with a few calculations below, changing it
         # may affect stability of the controller.
+        lookahead = 10
```

Comment:
I _think_ this change is safe since when I audited the following code it seems the most `Waypoints` we use here is 10 anyway.

Reply:
> I _think_ this change is safe since when I audited the following code it seems the most `Waypoints` we use here is 10 anyway.

Nope.  It did matter.  All the wavepoints are used in the road curviness computation.  10 is not enough.  (14 seems to be, but reverting it back to 16 just in case.)